lib/galaxy/selenium/navigation.yml

```diff
@@ -321,10 +321,10 @@ edit_dataset_attributes:
 
 edit_collection_attributes:
   selectors:
-    database_genome_tab: 
+    database_genome_tab:
       type: xpath
       selector: '//a[contains(text(), "Database/Build")]'
-    database_value: 
+    database_value:
```

Comment:
Looks like automatic whitespace-line-end trimming, yep.  Mine's set up to do that too, looks okay?

Reply:
In my opinion, automatic whitespace-line-end trimming of unrelated code paths shouldn't be a part of this PR. If we really want to enforce yml style, it should be a separate pull request. Otherwise it will just create noise in gitlens and will make an impression that changed lines are related to username.

Reply:
I can rewrite that commit, yeah.

Reply:
@OlegZharkov Separate commit makes sense for sure, separate PR, at least for this, I'd say is overly cumbersome.  Git blame can show a "Cleaned up whitespace" commit.